lib/actions/device.coffee

```diff
@@ -272,7 +272,7 @@ exports.init =
 
 			(callback) ->
 				return callback(null, options.application) if options.application?
-				vcs.getApplicationName(process.cwd(), callback)
+				vcs.getApplicationName(process.cwd(), callback).nodeify(callback)
```

Comment:
There is no need to pass the callback as an argument to the function, since it's being handled by `.nodeify()`.